pynucastro/rates/library.py

```diff
@@ -625,7 +625,7 @@ def backward(self):
         only_bwd = self.filter(only_bwd_filter)
         return only_bwd
 
-    def forward_for_detailed_balance(self):
+    def forward_for_detailed_balance(self, use_unreliable_spins=True):
```

Comment:
I could also be convinced to leave this off as a parameter, but it does affect the behavior of the function due to unreliable spins being able to throw an error, so I think it's best to expose that to the user (and more relevantly, to `derived_backward()`).